andela/ah-robotics-frontend
# #162949033 Setup Sass PreProcessing

#### What does this PR do?

Adds sass preprocessing to the application to enable use of both .scss and css stylesheets

#### Description of Task to be completed?

- [x] install node-sass npm package
- [x] change .css extensions to .scss

#### How should this be manually tested?

1. Start terminal application and clone this repository on your local machine using: 
`git clone https://github.com/andela/ah-robotics-frontend.git`
2. Change your working directory to the project's directory using: 
`cd ah-robotics-frontend`
3. Change to the specific branch `ch-sass-integration-162949033` using: 
`git checkout ch-sass-integration-162949033`

4. Run `npm install` to install all the modules listed in the `package.json` file.

5. Run `npm start` command to start the application.

6. The application should start successfully displaying a page as shown below. The only change is that this is using the .scss stylesheets instead of .css.
<img width="1440" alt="screen shot 2019-02-07 at 1 19 04 pm" src="https://user-images.githubusercontent.com/10683903/52404836-0b760880-2adb-11e9-82b8-533a52d32933.png">

#### What are the relevant pivotal tracker stories?

[#162949033](https://www.pivotaltracker.com/story/show/162949033)
